Tools/python/MetaSubmitter.py

```diff
@@ -289,14 +295,14 @@ def set_splitting(samples, dataset, jobtype='default', data_json=None, default_f
 ####
 
 def pick_samples(dataset, both_years=False,
-                 qcd=True, ttbar=True, all_signal=True, data=True, leptonic=False, bjet=False,
+                 qcd=True, ttbar=True, data=True, leptonic=False, splitSUSY=False, Zvv=False, met=False,
                  span_signal=False):
 
     if span_signal:
         print 'cannot use both span and all_signal, turning off the latter'
         all_signal = False
 
-    argnames = 'qcd', 'ttbar', 'all_signal', 'span_signal', 'data', 'leptonic', 'bjet'
+    argnames = 'qcd', 'ttbar', 'span_signal', 'data', 'leptonic', 'splitSUSY', 'Zvv', 'met'
     args = dict([(a,eval(a)) for a in argnames])
```

Comment:
Adding the met/splitSUSY/Zvv options look good, but I think you removed some others by mistake!

Reply:
I removed because I kind of reformed those options in Samples.py to make it suitable for MET case. But maybe should change it back

Reply:
Yeah, I think we should make it generic, since we'll all (ideally) use the same base code.